test/e2e/storage/persistent_volumes-local.go

```diff
@@ -490,19 +494,28 @@ func makeLocalPVConfig(config *localTestConfig, volume *localTestVolume) framewo
 func createLocalPVCPV(config *localTestConfig, volume *localTestVolume) {
 	pvcConfig := makeLocalPVCConfig(config)
 	pvConfig := makeLocalPVConfig(config, volume)
-
 	var err error
 	volume.pv, volume.pvc, err = framework.CreatePVPVC(config.client, pvConfig, pvcConfig, config.ns, true)
 	framework.ExpectNoError(err)
 	framework.ExpectNoError(framework.WaitOnPVandPVC(config.client, config.ns, volume.pv, volume.pvc))
 }
 
 func makeLocalPod(config *localTestConfig, volume *localTestVolume, cmd string) *v1.Pod {
-	return framework.MakePod(config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, cmd)
+	return framework.MakeSecPod(config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, cmd, false, false, selinuxLabel)
```

Comment:
does this place additional requirements on CI envs? what happens on OSes that don't have selinux support or have it disabled?

Reply:
I run this test with SELinux disabled and I have not seen any impact so I do not expect any additional requirements for CI. If there is a way to test it on CI, please let me know how, I would definitely give it a spin there.

Reply:
cc @kubernetes/sig-storage-pr-reviews on SELinux CI question

Reply:
I'll run this PR against my GCE env and see.

Reply:
I built and ran this PR against my GCE e2e env and all the tests passed.